Dockerfile

```
@@ -16,4 +20,4 @@ VOLUME ["/root/.stremio-server"]
 
 EXPOSE 11470
 
-ENTRYPOINT [ "node", "server.js" ]
+ENTRYPOINT sh /stremio/start.sh
```

Maybe this can be simplified even more, would appreciate if you can test this though.
```suggestion
ENTRYPOINT sh start.sh
```